From 9d8c53e7b79c4bf95dc225dc5df88a70d95ae326 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Thu, 9 Aug 2018 18:30:01 +0300
Subject: [PATCH 0843/1200] pcie: mvebu: add pci=pcie_bus_safe to default
 kernel bootargs

setting 'pci=pcie_bus_safe' will set every PCIe device's MPS to the
largest value supported by all devices below the root complex,
This settings leads to optimized PCIe MPS size, and increases
performance for supported PCIe cards.

added extra_params=pci=pcie_bus_safe to the kernel boot string.

Change-Id: I8988cbaf6c8dd992bdc047ee07a8f91f34812bae
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/58942
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 include/configs/mvebu_armada-common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index d6ac8731dd..cc8b99d055 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -39,6 +39,7 @@
 						"booti $kernel_addr_r " \
 						"$ramdisk_addr_r " \
 						"$fdt_addr_r\0" \
+					"extra_params=pci=pcie_bus_safe\0" \
 					"kernel_addr_r=0x7000000\0"	\
 					"initrd_addr=0xa00000\0"	\
 					"initrd_size=0x2000000\0"	\
-- 
2.22.0

